src/Form/index.tsx

```diff
@@ -341,6 +390,13 @@ function Form({
 
     setAutoValidate(false); // Each step to initially not auto validate
 
+    if (
+      contextRef &&
+      Object.prototype.hasOwnProperty.call(contextRef, 'current')
+    )
+      // Need to update the context because it has activeStep closure'd in
+      contextRef.current = getFormContextHelper();
```

Comment:
should this actually be called in the getNewStep fucntion before the onLoad() so the user has these functions available ASAP

Reply:
oh yeah I can just call it there and pass in newStep instead of needing activeStep to be set